### ISSUE TITLE
load the pretrained identity enconder

### ISSUE THREAD

**Issue by yangyangma97** [chenhao2345/GCL#14]
Couldyou please tell me whether the pretrained identity encoder network you are training in stage 2 is idnet or gen_ pth\dis_.th？  For this problem, I can modify the K value without reporting an error, but I feel that it will affect the training accuracy.

**Comment by chenhao2345** [comment]
Please first check you are working on Pytorch 1.2. Some people said newer Pytorch version might raise a problem, because of the updated syntax.

In stage 2, we pre-train generator and discriminator. The identity encoder is frozen, but is still saved. After stage 2, you should have saved weights like this.

![image](https://user-images.githubusercontent.com/67144451/148396958-a7c65274-c79a-4bab-a17d-bd602734ecfa.png)
